Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public class AudioManager : MonoBehaviour
+{
+    private static AudioManager current;
+
+    public bool DebugPlayOnStart = false;
+    
+    [Header("Audio FX")]
+    [SerializeField] private float fadeDuration = 1f;
+    [SerializeField] private float initialCutoff = 22000f;
+    [SerializeField] private float muffledCutoff = 780f;
+    [SerializeField] private float tenseMusicSpeed = 1.1f;
+    
+    [Header("Audio Clips")]
+    [SerializeField] private AudioClip ambientClip;
+    [SerializeField] private AudioClip mainMusicClip;
+    [SerializeField] private AudioClip tenseClip;
+    [SerializeField] private AudioClip winClip;
+    [SerializeField] private AudioClip loseClip;
+    
+    [Header("Audio Mixer")]
+    [SerializeField] private AudioMixer mainMixer;
+    [SerializeField] private AudioMixerGroup ambientGroup;
+    [SerializeField] private AudioMixerGroup musicGroup;
+
+    private AudioSource ambientSource;
+    private AudioSource musicSource;
+
+    private void Awake()
+    {
+        // Keep 1 instance of the Audio Manager at all times.
+        if (current != null && current != this)
+            Destroy(gameObject);
+
+        current = this;
+        DontDestroyOnLoad(gameObject);
+        
+        // Generate the Audio source channels.
+        this.ambientSource = gameObject.AddComponent<AudioSource>() as AudioSource;
+        this.musicSource = gameObject.AddComponent<AudioSource>() as AudioSource;
+
+        this.ambientSource.volume = 0f;
+        this.musicSource.volume = 0f;
+        
+        this.ambientSource.outputAudioMixerGroup = ambientGroup;
+        this.musicSource.outputAudioMixerGroup = musicGroup;
+
+        // Set initial values.
+        this.mainMixer.GetFloat("musicCutoff", out initialCutoff);
+        this.musicSource.pitch = 1f;
+        
+        this.ambientSource.clip = this.ambientClip;
+        this.ambientSource.loop = true;
+        this.ambientSource.Play();
+        StopAmbientAudio();
```

Comment:
Why do you successively play and stop the audio?

Reply:
Honestly, I don't remember why I put that there 😂
Good catch